plots/plot_results_in_box_plot.py

```diff
@@ -0,0 +1,204 @@
+"""Plot a heatmap plot showing the accuracy of lots of time series prediction methods 
```

Comment:
It would be prudent to update the description to mention boxplots

Reply:
Done